frontend/pages/component/NavBar.tsx

```diff
@@ -19,7 +23,11 @@ const navigation = [
 
 export default function NavBar() {
     const [sidebarOpen, setSidebarOpen] = useState(false);
-
+    const [darkTheme, setDarkTheme] = useState(isDarkTheme());
+    const mainToggleTheme = () => {
+        const newTheme = toggleTheme();
+        setDarkTheme(newTheme);
+    };
```

Comment:
Remove this and the useState and just use 'toggleTheme' and 'isDarkTheme' from the import